pkg/webservice/handlers_test.go

```diff
@@ -318,6 +318,18 @@ func newApplication(appID, partitionName, queueName, rmID string, ugi security.U
 	return objects.NewApplication(siApp, userGroup, nil, rmID)
 }
 
+func TestGetStackInfo(t *testing.T) {
+	req, err := http.NewRequest("GET", "", strings.NewReader(baseConf))
+	assert.NilError(t, err, "Error creating request")
+	resp := &MockResponseWriter{}
+	getStackInfo(resp, req)
+
+	// assert stack trace
+	assert.Assert(t, strings.Contains(string(resp.outputBytes), "goroutine"), "Stack trace should be present in the response")
+	assert.Assert(t, strings.Contains(string(resp.outputBytes), "test"), "Stack trace should be present in the response")
+	assert.Assert(t, strings.Contains(string(resp.outputBytes), "github.com/apache/yunikorn-core/pkg/webservice.getStackInfo"), "Stack trace should be present in the response")
+}
```

Comment:
Can you cover -ve cases?

Reply:
Is there a negative case for getStackInfo? It seems to be cannot make runtime.Stack() failed. 

Ps. I added a deep stack trace case.

Reply:
I meant, asserting cases like StatusInternalServerError etc by creating such situation.

Reply:
@manirajv06 Just to clarify, I'm actually asking how to create a negative case (such as StatusInternalServerError). The whole API doesn't need any parameter, and the function mainly use `runtime.Stacktrace`, not sure if there's a way to fail it...

Reply:
StatusInternalServerError is returned only when go built in write() method fails. Yes, not required to cover the case. Just to be safer, we can assert status code against valid response code (200).

Reply:
Got it, done.